src/subdomains/monitoring/observers/staking-combined.observer.ts

```diff
@@ -82,8 +82,8 @@ export class StakingCombinedObserver extends MetricObserver<StakingData> {
     const dbBalance = await getCustomRepository(StakingRepository)
       .createQueryBuilder('staking')
       .leftJoin('staking.asset', 'asset')
-      .where('asset.name = :name', { name: 'DFI' })
       .select('SUM(balance)', 'balance')
+      .where('asset.name = :name', { name: 'DFI' })
```

Comment:
didn't know that they can be written in any order with this builder. for me it would make sense to order it like a normal SQL
SELECT, JOIN, WHERE, ORDER

Reply:
Yes agree. I at least wanted to have the SELECT above the WHERE. There was a place where we forgot to use a andWhere because there was a SELECT in between.

Reply:
However, IMO here it makes sense to have the joins at the top, because `createQueryBuilder` acts as a FROM.